scheduled-jobs-alert.tf

```diff
@@ -0,0 +1,62 @@
+module "blob-dispatcher-alert" {
+  source            = "git@github.com:hmcts/cnp-module-metric-alert"
+  location          = "${azurerm_application_insights.appinsights.location}"
+  app_insights_name = "${azurerm_application_insights.appinsights.name}"
+
+  enabled    = "${var.env == "prod"}"
+  alert_name = "Reform_Blob_Dispatcher_-_BSP"
+  alert_desc = "Triggers when no logs from blob-dispatcher job found within timeframe."
+
+  app_insights_query = "traces | where message startswith 'Started blob-dispatcher job'"
+
+  frequency_in_minutes       = 10
+  time_window_in_minutes     = 10
```

Comment:
Probably best to change it to something like 30 minutes maybe (definitely not less than 15). There's a 10 minute delay between consecutive job runs, so there's an even greater gap between start times.